recipes/r-stitch/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set version = "1.5.4" %}
+{% set sha256 = "a89a8b7d4bb70c9fb7fdbc71f5198a0f1718f7fe5141a2fccce9995998050fd9" %}
+
+package:
+  name: r-stitch
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/rwdavies/stitch/releases/download/{{ version }}/STITCH_{{ version }}.tar.gz
+  sha256: '{{ sha256 }}'
+
+build:
+  number: 0
+  script: '$R CMD INSTALL --build --install-tests .'
+
+requirements:
+  build:
+  host:
+    - r-base
+    - r-rcpp >=0.12.18
+    - r-rcpparmadillo >=0.8.600.0.0
+    - r-data.table >=1.11.8
+    - r-rrbgen >=0.0.6
+    - xz >=5.2.2
```

Comment:
If this is a dependency do you also need compilers added?
If you like you can apply this against the bulk branch - this will compile this recipe against GCC7. We are currently rebuilding all R packages against these new compilers.